dom0/sd-clean-all.sls

```diff
@@ -12,9 +12,7 @@ remove-dom0-sdw-config-files:
     - names:
       - /opt/securedrop
       - /etc/yum.repos.d/securedrop-workstation-dom0.repo
-      - /usr/bin/securedrop-update
       - /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test
-      - /etc/cron.daily/securedrop-update-cron
```

Comment:
Because the clean action ensures state=absent, it actually makes sense to preserve the entries here. Note that `/srv/*` items may still need to be removed, either here or via `clean-salt`.

Reply:
Done.